cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -29,16 +34,20 @@
                               search_types=settings.search_types,
                               search_size=settings.search_size)
 %}
+{% set hide_header_slug = settings.hide_header_slug or false %}
```

Comment:
Maybe make `hide_header_slug` it's own argument so it doesn't need to be pulled out of `settings`?

```
{% macro render(settings, hide_header_slug=false) %}
```